tools/reduce/src/main.rs

```diff
@@ -17,12 +17,12 @@ use std::{
     io::Write,
     os::unix::prelude::PermissionsExt,
     path::{Path, PathBuf},
-    process::Command,
+    process::Command as Cmd
```

Comment:
I think I'd prefer that this line be removed, and instead `std::process::Command` is spelled out explicitly where it's used in the code, if that's OK.